### PR TITLE
(gobby) Migrated from Repo

### PR DESCRIPTION
**Pull request by AdmiringWorm**
Gobby package migrated from [my repository](https://github.com/AdmiringWorm/chocolatey-packages)

### CODE REVIEW

File: automatic/gobby/gobby.nuspec [hunk 1]
```diff
@@ -0,0 +1,60 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “O” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <!-- Read this before publishing packages to chocolatey.org: https://github.com/chocolatey/chocolatey/wiki/CreatePackages -->
+    <id>gobby</id>
+    <title>Gobby</title>
+    <version>0.5.0</version>
+    <authors>aburgm</authors>
+    <owners>AdmiringWorm</owners>
```
Comment (gep13) [inline automatic/gobby/gobby.nuspec]: @AdmiringWorm can you add the chocolatey user in here as well?

Comment (AdmiringWorm) [inline automatic/gobby/gobby.nuspec]: sure, space separated right?

**Comment by gep13** [comment]
Now, you are asking... two secs...

**Comment by gep13** [comment]
Yes, space separated.

**Comment by AdmiringWorm** [comment]
updated, that's one place nuget and chocolatey differs.
According to the latest nuget docs, they are now comma separated

**Comment by gep13** [comment]
hold on...

**Comment by AdmiringWorm** [comment]
I remember it was space separated in the past though.

**Comment by gep13** [comment]
https://docs.nuget.org/ndocs/schema/nuspec#optional-metadata-elements

We don't deviate from this, I don't think.

The output of `choco new` doesn't cover this, and perhaps it should.

@ferventcoder thoughts?

**Comment by AdmiringWorm** [comment]
@gep13 I see that existing packages uses space:
https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/automatic/freecad/freecad.nuspec#L7

I think comma separated owners list was added in perhaps version 3+

**Comment by gep13** [comment]
Hmmm...

Either way, we have a validation rule for Tags:

https://github.com/chocolatey/package-validator/blob/master/src/chocolatey.package.validator/infrastructure.app/rules/TagsAreSpaceSeparatedRequirement.cs

I think we should have one for owners as well, once we decide what it "should" be 😄

**Comment by AdmiringWorm** [comment]
@gep13 The [AuthorDoesNotMatchMaintainerNote](https://github.com/chocolatey/package-validator/blob/master/src/chocolatey.package.validator/infrastructure.app/rules/AuthorDoesNotMatchMaintainerNote.cs) validator looks like it jons to comma separated values for both owners and authors

**Comment by gep13** [comment]
@AdmiringWorm ah, good catch!

In which case, let's go with comma separated then 😄

**Comment by AdmiringWorm** [comment]
gotcha, updated.

**Comment by AdmiringWorm** [comment]
@gep13 Any updates on this?

**Comment by gep13** [comment]
LGTM! Thanks again!

**Comment by AdmiringWorm** [comment]
deja vu, I think you said that when I first pushed the package to chocolatey as well XD

**Comment by gep13** [comment]
>@AdmiringWorm said...
deja vu, I think you said that when I first pushed the package to chocolatey as well XD

I meant it then, and still mean it 😄

**Pull request merged**